testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/adapter/example/AbstractJSConsoleExampleAdapterTest.java

```diff
@@ -351,6 +353,42 @@ public void loginRequiredAction() {
         waitUntilElement(jsConsoleTestAppPage.getOutputElement()).text().contains("Init Success (Authenticated)");
     }
 
+    @Test
+    public void testUpdateToken() {
+        logInAndInit("standard");
+
+        jsConsoleTestAppPage.setTimeSkewOffset(-33);
+        setTimeOffset(33);
```

Comment:
timeSkew is an internal detail in keycloak.js and it's not correct to set it directly to force an update of the token. It would be better to increase the minValidity and/or reduce token expiration for the realm.